### PR TITLE
Blog: Covid announcement

### PR DESCRIPTION
**Pull request by wordshaker**
## Description of change
A post about our decision about the 2020 events

## Authors
@wordshaker Jessica White

### CODE REVIEW

File: _posts/2020-06-21-covid-decision.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Will The Conference Go Ahead In 2020?
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:3]: I think the title should make it clear the event is postponed and the description should be why you have come to that decision. Having to click through to find the answer could be a bit frustrating.

File: _posts/2020-06-21-covid-decision.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+The team met every 4 weeks to assess the statistics around COVID-19 and what it would take to run a safe and successful conference. As two of our team are currently shielding, this also included the ability or organise the conference effectively.
+
+It looks likely that some form of social distancing will still be in place by the time October comes around. Not only is the overhead of ensuring social distancing is adhered to a logistical challenge, but we think it will have a big effect on the feel of the conference itself. 
+
+DDD East Midlands is a chance for the community and local businesses to celebrate and showcase all that makes the local tech community great. There are a range of activities planned for the conference and a lot of this includes some range of closeness or contact with surfaces. 
+
+The health and safety of our delegates, and that they have a fantastic experience of the event is really important to us.
+
+### The other options we considered.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:35]: Consider re-wording to something like "Why we're not moving online"? as this is the only other option discussed

File: _posts/2020-06-21-covid-decision.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+We understand there will be a lot of questions around this. Hopefully, the below will answer a few of them. As always, we are here to answer any others you may have.
+
+### What day is DDD East Midlands 2021?
+
+Saturday 2nd October 2021.
+
+### Can I carry over the ticket I have for 2020?
+
+Yes. We will be closing off the ability to sign up for a ticket until next year, but any non-cancelled tickets will be carried over to 2021.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:59]: I don't think you need to specify non cancelled tickets here

File: _posts/2020-06-21-covid-decision.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+### Will you be pausing ticket sales.
+
+Yes. Though tickets are being carried over, we will close off tickets for now while we plan for 2021.
+
+### What happens for speakers?
+
+We are still hosting the online virtual workshop for the speakers selected for 2020, but we have decided to re-start the talk submission and voting process for 2021. 
+
+There are a couple of reasons for this. A lot can change in a year, and we know that often speakers like to change their topics and talks in this time frame. Further to this we want to have an opportunity for more people to put forward their submissions. It is part of our mission to support and empower people to speak at events like ours, and we are going to continue to offer opportunities to take part.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:69]: Might be worth adding that some of the talks might not be relevant in a years time

File: _posts/2020-06-21-covid-decision.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+Yes. We will be closing off the ability to sign up for a ticket until next year, but any non-cancelled tickets will be carried over to 2021. 
+
+### Will you be pausing ticket sales.
+
+Yes. Though tickets are being carried over, we will close off tickets for now while we plan for 2021.
+
+### What happens for speakers?
+
+We are still hosting the online virtual workshop for the speakers selected for 2020, but we have decided to re-start the talk submission and voting process for 2021.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:67]: It reads that submissions for 2021 are open for talks? Could change to "..but we have decided we will re-start the..." so it's clear submissions aren't open yet for 2021

File: _posts/2020-06-21-covid-decision.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+We will use some of this time to expand the range of audiences we reach and to continue to encourage people to try speaking. Throughout we will offer support to those wanting to try speaking at local events, and submitting to the conference next year.
+
+Everyone has something valuable to say and educational content to share. There is a lot of talented and smart people throughout the East Midlands. We want to encourage them to take to the stage and further enrich the local communities tech scene.
+
+---
+
+# Thank you.
+
+From all the team, thank you for all the support you have shown the event, and to us. We have had a number of challenges in the last couple of years. We can't wait to host a fantastic event that hopefully you will all submit to, attend and enjoy.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:124]: Could re word the last sentence to something a bit stronger? Something like "We can't wait to host a fantastic event that you can enjoy where everyone can submit to speak, attend and enjoy"

File: _posts/2020-06-21-covid-decision.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+<i>The DDD Flags outside Nottingham Conference Centre</i>
+</center>
+<br/>
+
+---
+<br/>
+
+# Common questions 
+
+We understand there will be a lot of questions around this. Hopefully, the below will answer a few of them. As always, we are here to answer any others you may have.
```
Comment (AnnaDodson) [inline _posts/2020-06-21-covid-decision.md:51]: Link to contact details or how they can get in touch?

File: _posts/2020-06-21-covid-decision.md (hunk 4 above)
Comment (wordshaker) [reply _posts/2020-06-21-covid-decision.md:69]: It's more that people tend not to be interested in, or want to talk about certain topics multiple years  in a row (plus we got hit by covid around the talk submission time 😢 which we think lead to less people submitting overall)

**Pull request merged**